### ISSUE TITLE
Autocomplete is not accessible and needs to be adjusted

### ISSUE THREAD

**Issue by alexwbbr** [Capgemini/dcx-react-library#624]
**Describe the bug**
The auto complete isn't very accessible to users using a screen reader, the auto complete needs to be implemented in a similar fashion to this: https://alphagov.github.io/accessible-autocomplete/examples/#autocomplete-minLength
